src/main/resources/view/ContactListBox.fxml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.HBox?>
+<HBox id = "box" fx:id="box" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+    <HBox fx:id="emailAddressContainer" HBox.hgrow="ALWAYS"/>
+    <HBox fx:id="twitterHandleContainer" HBox.hgrow="ALWAYS"/>
+    <HBox fx:id="telegramHandleContainer" HBox.hgrow="ALWAYS"/>
```

Comment:
Contact should be added dynamically into the `HBox`. Perhaps we can move the `adding` logic into `ContactListBox.java`